src/components/Icon/custom/add-patient.svg

```diff
@@ -0,0 +1,4 @@
+</svg>
```

Comment:
Not a request for changes but more a question. Is the fact that there are no newlines at the end of the svg files a potential problem?

Reply:
Hey Blessed,

There were no errors either with the tests or with the rendering in Storybook with no newlines. However, I can make that slight change to all the svg files if necessary. Thanks!

Reply:
Hey @blestab,

Just added the newlines at the end of the SVG files just in case. Thanks!